src/github.com/matrix-org/dendrite/syncapi/consumers/roomserver.go

```diff
@@ -28,15 +28,15 @@ import (
 	sarama "gopkg.in/Shopify/sarama.v1"
 )
 
-// Server contains all the logic for running a sync server
-type Server struct {
+// Roomserver consumes events that originated in the room server.
+type Roomserver struct {
```

Comment:
Hmm, I guess you refer to it by `consumers.Foo` everywhere so it might be clear enough without suffixing with `Consumer`.

However I still dislike that it's named after the thing writing the event rather than the actual event it consumes.
It shouldn't care which sever is writing to that stream, it should only care about the contents.

(In this case there will only ever be one server writing to that stream, but I feel it's a dangerous precedent to set in the codebase)